text/survey_analysis_report2.md

```diff
@@ -0,0 +1,219 @@
+
+## Issues
+
+Most participants could see the advantages of having a standardized structure for their files,
+but were critical about the **cost to benefit ratio** of the process, especially for ongoing projects.
+Many mentioned that it would only reach full impact if the whole lab would be using it, emphasizing the advantage of such a system for
+collaborative work.
+In addition, they mentioned the time saved by not having to create a template for themselves,
+but only use an existing standard.
+While they were quite unanimous about using a template for new projects, they were skeptical about making a transition for ongoing projects,
+as the cost of transition might be too high.
+More generally, they were worried that learning a new workflow to use the template could be time consuming.
+
+
+In many cases, people mentioned that the **files are organized via experiments**, not at the project level.
+In particular, people tend to pool data and code in a single experiment folder.
+This is reminiscent of the two example structures given in the library carpentry course [@LibraryCarpentryLcfairresearch2019].
+
+
+On the other hand, some **files are organized outside of project folders**, that is in particular places irrespective of the project they belong to. 
+Many researchers reported having a folder for all conference reports or all manuscripts, for instance.
+
+
+# The new template
+
+## Template overview
+
+Data management principles recommend (1) to keep all files related to a project in a single folder (this facilitates sharing of these files with the whole team working on the project, for instance),
+and (2) to manage data and code differently (this allows different version control system, as well as independent sharing and reusing of data and code).
+We designed a template that follows these principles, 
+but added some recommendations and technical solutions 
+in order to permit users to have laboratory and experiment-level organization of their files, nevertheless.
+
+![Fig.4: Folder structure developed after analyzing the survey. The template works mostly on the project level (one unique folder for all files related to one project). The experiment level is taken care by specifying several experiment sub-folders when new experiments are started. By sharing specific subfolder independently in a cloud solution, one can reorganize information in cross-project directories that host sub-folders coming from different projects. Note that both the creation of experiment sub-folders and the creation of laboratory level organization is automated in GIN-tonic.](../50_figures/pulication_ready_figures/fig4.png)
+
+```{}
+Our definition of the organisation levels:
+
+- Research project: The unit of research that address a specific research question. It can mostly be deliminated by the team involved and typically produce a unique research paper.
```

Comment:
```suggestion
- Research project: The unit of research that address a specific research question. It can mostly be deliminated by the team involved and typically produces a unique research paper.
```